.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,88 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/python-3
+{
+	"name": "Python 3",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": "..",
+		"args": {
+			// Update 'VARIANT' to pick a Python version: 3, 3.6, 3.7, 3.8, 3.9
+			"VARIANT": "3.9",
+			// Options
+			"NODE_VERSION": "none"
+		}
+	},
+	"runArgs": [
+        // Enable GPU passthrough, requires WSL2 on Windows
+		//"--gpus=all",
+	],
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"editor.formatOnSave": true,
+		"editor.rulers": [
+			120
+		],
+		"python.pythonPath": "/usr/local/bin/python",
+		"python.defaultInterpreterPath": "/usr/local/bin/python",
+		"python.languageServer": "Pylance",
+		"python.autoComplete.addBrackets": true,
+		"python.analysis.autoImportCompletions": true,
+		"python.analysis.completeFunctionParens": true,
+		"python.analysis.autoSearchPaths": true,
+		"python.analysis.useImportHeuristic": true,
+		"python.sortImports": true,
+		"python.sortImports.args": [
+			"--profile",
+			"black",
+			"--atomic"
+		],
+		"python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
+		"python.formatting.blackPath": "/usr/local/py-utils/bin/black",
+		"python.formatting.provider": "black",
+		"python.formatting.blackArgs": [
+			"--line-length=120"
+		],
```

Comment:
Is it possible to use the project configuration instead of providing the specific value?
https://github.com/PyTorchLightning/metrics/blob/1fef5aef6e97c1dec3ed04e9f26e9e665a448e74/pyproject.toml#L7-L10